src/main/java/be/uantwerpen/scicraft/block/AtomicFloor.java

```diff
@@ -9,14 +9,19 @@
 import net.minecraft.util.shape.VoxelShapes;
 import net.minecraft.world.BlockView;
 
-public class AtomicFloor extends Block {
-//    private static QuantumFieldSpawner generator;
+public class AtomicFloor extends AbstractGlassBlock {
+    public final static int AtomicFloorLayer = 64;
```

Comment:
Hard to see in a webview but this seems to be only used inside QuantumFieldSpawner, maybe we should move it there. 
Also not really sure about this if this is the most clean way of doing things. When we ever change the layers inside the subatomic dimension we have to change this too, which is kinda against best practices.

Reply:
Makes me think about the way the despawning works right now, should be moved to the quantumfield class
i will do that now

Reply:
I moved the breaking to scheduledTick, i could move the atomicFloorLayer to QuantumfieldBlock but it feels more logical to store it inside atomic floor. Or we could store it in some general purpose static variable file.

Reply:
Ideally it would be more connected to the dimension, so when we lower or raise the floor layer in the dimension layers we wouldn't have the change the value here too, which is not the case. If other mods try to use AtomicFloor the hardcode will be problematic.

Reply:
I approved changes for now, but it wouldn't be bad to have someone else look at this for an extra opinion.